src/Test/L0/Worker/TestResults/TestRunPublisherTests.cs

```diff
@@ -352,6 +352,41 @@ public void PublishConsoleLogIsSuccessful()
             Assert.Equal(decodedData, testResultWithLog.ConsoleLog);
         }
 
+        [Fact]
+        [Trait("Level", "L0")]
+        [Trait("Category", "PublishTestResults")]
+        public void PublishStandardErrorLogIsSuccessful()
+        {
+            SetupMocks();
+            ResetValues();
+
+            TestCaseResultData testResultWithStandardError = new TestCaseResultData();
+            testResultWithStandardError.StandardError = "Publish standard error is successfully logged";
+
+            TestCaseResultData testResultWithNoStandardError = new TestCaseResultData();
+            testResultWithNoStandardError.StandardError = "";
+
+            TestCaseResultData testResultDefault = new TestCaseResultData();
+
+            List<TestCaseResultData> testResults = new List<TestCaseResultData>() { testResultWithStandardError, testResultWithNoStandardError, testResultDefault };
```

Comment:
Can you add one more test case where multiple testResultsWithStandardError?

Reply:
Added